spec/content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/used_spec.rb

```diff
@@ -71,12 +71,60 @@
     it_behaves_like "used"
   end
 
+  shared_examples_for "used user quota" do
+    let(:quota_source) { MiqAeMethodService::MiqAeServiceUser.find(@user.id) }
+    let(:quota_source_type) { 'user' }
+    let(:active_method) { 'active_provisions_by_owner'.to_sym }
+    it "check" do
+      @miq_provision_request.options[:owner_email] = owner_email
+      @user.email = requester_email
+      @miq_provision_request.save
+      @user.save
+      expect(svc_miq_request).to receive(:check_quota).with(active_method).and_return(active_counts_hash)
+      described_class.new(ae_service).main
+      expect(ae_service.root['quota_used']).to include(result_counts_hash)
```

Comment:
Please add a new-line between the test setup and the `expect`.

Reply:
Changed